import { errorHandler } from "../utils/error.JS";

export const sendEmail = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
